packages/next-on-pages/templates/_worker.js/utils/images.ts

```diff
@@ -66,7 +66,11 @@ export function getResizingProperties(
 		return undefined;
 	}
 
-	const isRelative = rawUrl.startsWith('/') || rawUrl.startsWith('%2F');
+	const isProtocolRelative =
+		rawUrl.startsWith('//') || rawUrl.startsWith('%2F%2F');
+	const isRelative =
+		(rawUrl.startsWith('/') || rawUrl.startsWith('%2F')) && !isProtocolRelative;
+
```

Comment:
i feel like it could be worth changing this to be a nicer regex instead now that it's getting bigger. what do you think? maybe something like `/^(%2F|\/)+/`